src/release_artifacts_resources/ios/cdk/cdk/credential_rotation/lambda_functions/src/source_data_generator/static_data_source.py

```diff
@@ -0,0 +1,12 @@
+import os
```

Comment:
Quibble (not a "complaint", but more than a "nit") -- this isn't static data, it's being retrieved from a Lambda env var. Maybe `lambda_env_var_data_source.py` and `retrieve_lambda_env_var_value`?

Unless I'm misunderstanding the purpose?

Reply:
yep, changed